### DISCUSSION TITLE
Property erros does not exist on type FormStateProxy

### DISCUSSION THREAD

**Discussion by minnyww** [react-hook-form/react-hook-form#5861] [Q&A]
version 
- react-hook-form : 7.10.1
- typescript : 4.3.5

how to fix this 

![image](https://user-images.githubusercontent.com/16731866/124709043-3e7b8b00-df25-11eb-9e3d-faf1a2a8c3b5.png)

**Comment by bluebill1049** [comment]
can you share a codesandbox?
